docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+
+All guides, whenever possible, demonstrate the steps from both kubectl and UI.
+
+## Prerequisites
+
+1. Install:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (1.16 or greater)
+- [curl](https://github.com/curl/curl)
+- [Kyma](/root/kyma/#installation-install-kyma-on-a-cluster) on a cluster of your choice.
+
+2. Download the `kubeconfig` file with the cluster configuration:
+
+a. Access the Console UI of your Kyma cluster.
+
+b. Click the user icon in the upper right corner.
+
+c. Select **Get Kubeconfig** from the drop-down menu to download the configuration file to a selected location on your machine.
+
+d. Open a terminal window.
+
+e. Export the **KUBECONFIG** environment variable to point to the downloaded `kubeconfig`. Run this command:
+
+   ```bash
+   export KUBECONFIG={KUBECONFIG_FILE_PATH}
+   ```
+
+   >**NOTE:** Drag and drop the `kubeconfig` file in the terminal to easily add the path of the file to the `export KUBECONFIG` command you run.
+
+f. Run `kubectl cluster-info` to check if you are connected to the correct cluster.
+
+## Main actors
+
+Let's introduce the main actors that will lead us through the guides. These are our in-house examples, mock applications, and experimental services:
+
+- [`orders-service`](https://github.com/kyma-project/examples/tree/master/orders-service) is a sample microservice written in Go. It can expose HTTP endpoints that can be used to create and read basic order JSON entities. The service can run with either an in-memory database that is enabled by default or an external Redis database. You will see on its example how you can use Kyma to deploy your own microservice, expose it on HTTP endpoints to make it available for external services, and then bind it to an external database service (Redis).
+
+- [`orders-function`](https://github.com/kyma-project/examples/tree/master/orders-service/deployment/function.yaml) that is an equivalent of `orders-service` with the ability to retrieve order records and save data in it. Like the microservice, the Function can run with either an in-memory database or a Redis instance.
+
+- [Redis addon](https://github.com/kyma-project/addons/tree/master/addons/redis-0.0.3) is basically a bundle of two Redis services available in two plans: `micro` and `enterprise`. You will connect it to Kyma through Helm Broker and expose it in the Kyma cluster as an addon under Service Catalog. The Redis service represents an open-source, in-memory data structure store, used as a database, cache and message broker. For the purpose of these guides, you will use the `micro` plan with in-memory storage to demonstrate how it can replace the default memory of the microservice.
```

Comment:
```suggestion
- [Redis addon](https://github.com/kyma-project/addons/tree/master/addons/redis-0.0.3) is basically a bundle of two Redis services available in two plans: `micro` and `enterprise`. You will connect it to Kyma through Helm Broker and expose it in the Kyma cluster as an addon under Service Catalog. The Redis service represents an open-source, in-memory data structure store used as a database, cache, and message broker. For the purpose of these guides, you will use the `micro` plan with in-memory storage to demonstrate how it can replace the default memory of the microservice.
```
I changed the commas:
`data structure store, used as a database, cache and message broker.` -> `data structure store used as a database, cache, and message broker.`
 The first one, after store, could remain, but I believe it negatively influences readability, as there are lots of commas around this place.